packages/antd/src/components/DatePicker/index.js

```diff
@@ -0,0 +1,6 @@
+import dayjsGenerateConfig from 'rc-picker/lib/generate/dayjs';
```

Comment:
This does seem to be a better solution than using the day-js-webpack-plugin.

Was day-js-webpack-plugin causing #1880? If so, why?

Reply:
The `day-js-webpack-plugin` was not working as intended for some reason, meaning that the `DatePicker` component was still trying to access the default `moment` library instead of `dayjs` for date/time operations.